src/test/Test.java

```diff
@@ -0,0 +1,14 @@
+package test;
+
+import java.util.ArrayList;
+
+public class Test {
+
+	public static void main(String[] args) {
+		String s = new String("Hi");
+		ArrayList<String> b = new ArrayList<String>();
+		b.add(s);
+		b.remove(0);
+		System.out.println(s);
```

Comment:
TODO: Finish the `void removeDuplicate()` function, which is also a part of the homework!